manual/idempotence/README.md

```diff
@@ -0,0 +1,132 @@
+## Query idempotence
+
+A query is *idempotent* if it can be applied multiple times without changing the result of the initial application. For
```

Comment:
If queries are idempotent it shouldn't matter that they might get executed twice with different timestamps.
If they're not, and this matters to you, you should mark them as such and use the relevant policy that avoids retries.
So I think all in all it's fine.